solara/server/templates/solara.html.j2

```diff
@@ -27,7 +25,7 @@
     {{ resources.include_css("/static/highlight.css") }}
     {{ resources.include_css("/static/highlight-dark.css") }}
     {{ resources.include_css("/static/assets/style.css") }}
-    <style id="jupyter-theme-css">
+    <style id="jupyter-theme-css" class="solara-template-css">
         {{ resources.include_css("/static/assets/theme-"~theme.variant~".css") }}
```

Comment:
Do we want this content there? Since we immediately fill it using `changeThemeCSS`, and it might result in an extra flash?

Reply:
Good idea! I think that way we save one load of the stylesheet.